pkg/build/builder/sti.go

```diff
@@ -222,7 +222,15 @@ func (s *S2IBuilder) Build() error {
 		return errors.New(buffer.String())
 	}
 
-	// If DockerCfgPath is provided in api.Config, then attempt to read the the
+	// Resolve the runtime image reference to a docker pull spec if it has been
+	// specified and set the fields for s2i builder.
```

Comment:
this doesn't look like it's resolving anything...comment is wrong?  Seems like it assumes the RuntimeImage field has already been resolved to a pull spec by this point (hopefully that is a valid assumption?)